### ISSUE TITLE
Google  { error: "popup_closed_by_user" }

### ISSUE THREAD

**Issue by Exlord** [askidmobile/angular-6-social-login-v2#6]
I am getting this error in console after selection my user from the google signin page!

**Comment by jeanDenison** [comment]
Am facing the same problem. Were u able to fix it?

**Comment by Exlord** [comment]
Nope, I stopped working on that project so did not look for any solution

**Comment by vishuhanda001** [comment]
i think you need to clear cache before signing in using google
